### DISCUSSION TITLE
[4.x]: Need warning for duplicate slug when editing

### DISCUSSION THREAD

**Discussion by wbdzynr** [craftcms/cms#13027] [Ideas]
### Description
If you are editing an entry, and you make changes to the slug (for a desired result), it doesn't warn you if there is already an entry with that slug.  It just appends a number to the end again.

### Steps to reproduce
1. Duplicate an entry and try to remove the "-2" from the end, then save.  If you check entry #2, it will have a "-2" at the end of the slug again.

### Expected behavior
When creating a new entry, appending a number to the end of the slug is fine.  However, if you are editing and you expect a slug to be a certain way (ie you edited it) and that slug already exists, it would be nice if a warning would appear (ie "An entry with that slug already exists. (Entry ID: 111).") giving you a heads up that your desired changes won't happen.

### Actual behavior
Currently, it just appends a number to the end of the slug again.

### Craft CMS version

4.4.5

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
(Converted to a discussion as this is a feature request, not a bug with existing functionality.)
